tests/baselines/reference/contextuallyTypedBindingInitializerNegative.errors.txt

```diff
@@ -17,34 +12,33 @@ tests/cases/conformance/types/contextualTypes/methodDeclarations/contextuallyTyp
         show: (x: number) => string;
     }
     function f({ show: showRename = v => v }: Show) {}
-                       ~~~~~~~~~~
-!!! error TS2322: Type '(v: number) => number' is not assignable to type '(x: number) => string'.
-!!! error TS2322:   Type 'number' is not assignable to type 'string'.
+                                         ~
+!!! error TS2322: Type 'number' is not assignable to type 'string'.

```

Comment:
This one is a little bit hard to judge. It's not super clear where this `number` is coming from and what it's being compared to.

Reply:
The overall signature has the same problem, since it's all inferred, IMO.